app/models/video.rb

```diff
@@ -8,4 +9,13 @@ def self.search_by_title(search_term)
     return [] if search_term.blank?
     where("title ILIKE ?", "%#{search_term}%").order("title ASC")
   end
+  
+  def average_rating
+    return 0 if reviews.empty?
+    ratings = []
+    reviews.each do |review|
+      ratings << review.rating
+    end
+    ratings.sum / ratings.count
```

Comment:
had no idea that existed thanks! Clearly I have a lot of documentation reading to catch up on!